src/collections/_documentation/workflow/alerts-notifications/notifications.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Other Notifications
+sidebar_order: 2
+---
+
+Aside from Alerts, Sentry has other types of notifications, such as Workflow, Deploy, and Quota notifications. All these notifications are sent via email.
+
+## Workflow
+
+Workflow relates to activity involving user actions and state changes on issues and alerts. This includes things like resolution, assignment, comments, and regressions. By default, Sentry subscribes you to issues it deems you're participating in, based on signals. 
```

Comment:
"based on signals." feels like an abrupt end to the sentence (it made more sense when the sentence was "based on signals such as.."). can we bring back the previous sentence? or if you want to keep it split, we can just remove it and end with "participating in."